keychain/btcwallet.go

```diff
@@ -278,9 +279,10 @@ func (b *BtcWalletKeyRing) DerivePrivKey(keyDesc KeyDescriptor) (
 			// Now that we know the account exists, we can safely
 			// derive the full private key from the given path.
 			path := waddrmgr.DerivationPath{
-				Account: uint32(keyDesc.Family),
-				Branch:  0,
-				Index:   uint32(keyDesc.Index),
+				InternalAccount: uint32(keyDesc.Family),
```

Comment:
Does this field actually need to be set in this specific context?

Reply:
The wallet now derives based on `InternalAccount` rather than `Account`.